core/domain/app_feedback_report_domain_test.py

```diff
@@ -98,6 +97,48 @@
 }
 ANDROID_REPORT_INFO_SCHEMA_VERSION = 1
 WEB_PLATFORM_VERSION = '3.0.8'
+PLATFORM_CHOICE_ANDROID = app_feedback_report_models.PLATFORM_CHOICE_ANDROID
+REPORT_JSON: app_feedback_report_domain.AndroidFeedbackReportDict = {
+        'platform_type': 'android',
```

Comment:
Wrong indentation. In Python our indentataion level is 4 not 8.

Reply:
made indentation level 4.